.buildkite/asr-eval/run-tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -x
+
+# Start server
+python3 -m vllm.entrypoints.openai.api_server --model openai/whisper-large-v3 $@ &
```

Comment:
Rather than having a script to launch the server. `test_transcription_api_correctness.py` could use the `RemoteOpenAIServer` test utility to launch the server from the python code. This will help with the maintainability of the test case

Reply:
sure, I was just unsure whether this test should belong to pytest suite or buildkite scipts. Agree `RemoteOpenAIServer` will make things cleaner